PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,16 @@
+## Purpose
+Describe the feature or issue and link to the related issue. 
+If no issue has been opened about this, be sure to open an issue first to discuss the need for this PR.
+
+## Description
+Please include any relevant code samples or screen shots that may help to overview of this PR.
+Link to specific lines of code, or examples if you need to.
+
+## Checklist
+* [ ] - All specs pass locally with `crystal spec/`
+* [ ] - All specs are formatted with `crystal tool format spec/`
+* [ ] - All code is formatted with `crystal tool format src/`
+* [ ] - Inline documentation has been added and/or updated
+* [ ] - An issue already exists detailing the issue/or feature request that this PR fixes
+* [ ] - Lucky builds on docker with `./scripts/setup`
+* [ ] - All builds and specs pass on docker with `./scripts/test`
```

Comment:
Rather than have this + `crystal spec` could you leave just this one? Since if it doesn't pass then it won't pass on CI. What do you think?

Reply:
I just assumed if local is macOS and docker, then you're guaranteed to have ran it on 2 OS. Maybe that doesn't matter in this case?

Reply:
Oh I see. I think doing just `script/test` should cover it. For this project it is fairly simple to set up but for other Lucky projects like Avram it is a bit trickier. So asking people to set up locally and with Docker seems like too much. Especially since in the end someone could be developing on Linux anyway, in which case we still wouldn't know if it works on macOS. So far I haven't seen any Linux <-> Mac issues though so I think it is ok to stick with the Docker build

Reply:
fair enough!